README.md

```diff
@@ -1,18 +1,28 @@
 # compute-sdk-go
 
-Experimental Go SDK for building [Compute@Edge](https://www.fastly.com/products/edge-compute/serverless) applications with [TinyGo](https://tinygo.org/).
+Experimental Go SDK for building [Compute@Edge](https://www.fastly.com/products/edge-compute/serverless) applications with [Go](https://go.dev) (1.21+) and [TinyGo](https://tinygo.org/) (0.28.1+).
```

Comment:
Can we drop "experimental" yet?

Reply:
Almost certainly.  (Unless we want to leave it until we tag 1.0)